arches/app/media/css/accessibility.scss

```diff
@@ -1,3 +1,159 @@
+$search-tab-buttons-bg: #FFF;
+$general-txt: #000;
+$btn-primary: #2B4E6D;
+$btn-clear-filters: #15503F;
+$btn-danger-warning: #943725;
+$profile-txt-general: #595959;
+$debug: #FFF;
+$leftnav-active: #184E6C;
+$leftnav-active-pointer: transparent #FFF transparent transparent;
+$search-dropdown-pill-bg: #1C6615;
+$adv-search-facet-btn-txt: #000;
+
+a {
+    &.search-candidate-title,
+    &.search-candidate-link {
+        color: $search-links;
+        &:hover,
+        &:focus {
+            text-decoration: underline;
+        }
+    }
+}
+
+.search-type-btn, .search-type-btn-popup {
+    color: $search-tab-buttons-txt;
+}
+
+.search-type-btn.relative:hover,
+.search-type-btn.relative:focus,
+.search-type-btn-popup.relative:hover,
+.search-type-btn-popup.relative:focus {
+    text-decoration: underline;
+    background: $search-tab-buttons-bg;
+}
+
+.select2-default,
+.chosen-container-single .chosen-single,
+.search-listing-body,
+.map-sidebar-text,
+.search-export .instruction h4,
+.rr-splash-description,
+.time-wheel-instructions,
+.search-facet-item.header input::placeholder,
+.time-search-container #search-from-c .input-md::placeholder,
+.time-search-container #search-from-b .input-md::placeholder,
+.related-resource-graph-information-content,
+.relationships-list-item-type,
+.relationships-list-count,
+.report-print-date,
+.rp-tile-title,
+.rp-report-section-title,
+.related-resources-graph-controls-instructions,
+.related-resources-graph-controls span,
+.rp-card-section .rp-no-data,
+.related-reasources-search-container input::placeholder,
+.workbench-card-sidepanel-header:before {
+    color: $general-txt!important;
+}
+
+.btn-primary,
+.library-icon .bg-primary,
+.select2-container-multi .select2-choices .select2-search-choice {
+    background-color: $btn-primary;
+}
+
+.pagination>.active>a,
+.pagination>.active>span,
+.pagination>.active>a:hover,
+.pagination>.active>span:hover,
+.pagination>.active>a:focus,
+.pagination>.active>span:focus {
+    background-color: $btn-primary;
+    border-color: $btn-primary;
+}
+
+.profile-label-shim,
+.profile-default-message-panel,
+.search-listing-body,
+.account-tips,
+.hover-feature-metadata,
+.pagination > li > a.disabled,
+.graph-designer-header,
+.graph-type {
+    color: $profile-txt-general;
+}
+
+.debug-notice {
+    color: $debug;
+}
+
+.sidenav-sm {
+	.sidenav-menu {
+		.active-sub {
+			a {
+				background-color: $leftnav-active;
+                &:after {
+					display: inline-block;
+                    border-style: solid;
+                    border-width: 5px 10px 5px 0;
+                    border-color: $leftnav-active-pointer;
+                    content: " ";
+                    position: relative;
+                    top: calc(-50% - 11px);
+                    left: 20px;
+				}
+			}
+		}
+	}
+}
+
+.btn-mint {
+    background-color: $btn-clear-filters!important;
+}
+
+.btn-danger, .btn-warning {
+    background-color: $btn-danger-warning!important;
```

Comment:
There is a contrast issue between the text color and this background color: see Advanced search `And` and `Or` buttons: 
![advanced search facet contrast issue](https://github.com/archesproject/arches/assets/22161487/b110f5cc-63cd-481b-89c6-3ac961b58f13)

Reply:
Fixed by commit e9c5a53

Reply:
Hey @phudson-he, I'm still seeing the contrast issue for `and` / `or` on the Advanced search facets

Reply:
My change got removed when I did the merge earlier. Now fixed in commit 8f84556